src/main/scala/com.snowplowanalytics.snowplow/scalatracker/Tracker.scala

```diff
@@ -123,287 +123,296 @@ class Tracker(emitters: Seq[TEmitter], namespace: String, appId: String, encodeB
   }
 
   /**
-   * Track a Snowplow unstructured event
-   * Alias for `trackSelfDescribingEvent`
-   *
-   * @param unstructEvent self-describing JSON for the event
-   * @param contexts list of additional contexts
-   * @param timestamp optional user-provided timestamp (ms) for the event
-   * @return The tracker instance
-   */
+    * Track a Snowplow unstructured event
+    * Alias for `trackSelfDescribingEvent`
+    *
+    * @param unstructEvent self-describing JSON for the event
+    * @param contexts list of additional contexts
+    * @param timestamp optional user-provided timestamp (ms) for the event
+    * @return The tracker instance
+    */
   @deprecated("Use Tracker#trackSelfDescribingEvent instead", "0.4.0")
-  def trackUnstructEvent(
-    unstructEvent: SelfDescribingJson,
-    contexts: Seq[SelfDescribingJson] = Nil,
-    timestamp: Option[Timestamp] = None): Tracker =
+  def trackUnstructEvent(unstructEvent: SelfDescribingJson,
+                         contexts: Seq[SelfDescribingJson] = Nil,
+                         timestamp: Option[Timestamp] = None): Tracker =
     trackSelfDescribingEvent(unstructEvent, contexts, timestamp)
 
   /**
-   * Track a Snowplow self-describing event
-   *
-   * @param unstructEvent self-describing JSON for the event
-   * @param contexts list of additional contexts
-   * @param timestamp optional user-provided timestamp (ms) for the event
-   * @return The tracker instance
-   */
-  def trackSelfDescribingEvent(
-    unstructEvent: SelfDescribingJson,
-    contexts: Seq[SelfDescribingJson] = Nil,
-    timestamp: Option[Timestamp] = None): Tracker = {
-
-    val payload = new Payload()
-    payload.add("e", "ue")
-    val envelope = SelfDescribingData(SelfDescribingEventSchemaKey, unstructEvent.normalize)
-    payload.addJson(envelope.normalize, encodeBase64, "ue_px", "ue_pr")
-    track(completePayload(payload, contexts, timestamp))
-
-    this
-  }
+    * Track a Snowplow self-describing event
+    *
+    * @param unstructEvent self-describing JSON for the event
+    * @param contexts list of additional contexts
+    * @param timestamp optional user-provided timestamp (ms) for the event
+    * @return The tracker instance
+    */
+  def trackSelfDescribingEvent(unstructEvent: SelfDescribingJson,
+                               contexts: Seq[SelfDescribingJson] = Nil,
+                               timestamp: Option[Timestamp] = None): Tracker =
+    Try {
```

Comment:
I think that chance to get an exception here is very-very small. And even if we'll get it - it'll be in `AsyncEmitter`'s thread and won't be catched.

Reply:
Same for other `Try`s in `Tracker`.

Reply:
Important note: emitter's exceptions are supposed to be handled via [callbacks](https://github.com/snowplow/snowplow-scala-tracker/issues/58).

Reply:
Humm what is your suggestion here? Well scenario changed a little bit in the new release branch. For example [here)](https://github.com/aparra/snowplow-scala-tracker/blob/master/src/main/scala/com.snowplowanalytics.snowplow/scalatracker/Tracker.scala#L434) can I use the tracker?

Reply:
My suggestion here was just to remove these `try/catch`. I don't see what they can catch here, but I'd be happy to reconsider, maybe I'm missing something.

Reply:
Humm, well I need catch the error somewhere. So If I remove of public method I must put somewhere. So it is not clear for me when I should move to.